politeiawww/api/v1/api.md

```diff
@@ -1473,6 +1475,129 @@ Reply:
 }
 ```
 
+### `Proposal voting status`
+
+Returns the voting status for a single public proposal
+
+**Route:** `GET /V1/proposals/{token}/votingstatus`
+
+**Params:** none
+
+**Result:**
+
+| | Type | Description |
+|-|-|-|
+| proposalvoting | VotingStatus  | Proposal voting status |
+
+**VotingStatus:**
+
+| | Type | Description |
+|-|-|-|
+| token | string  | Censorship token |
+| status | int | Status identifier |
+| votessummary | CastedVotesSummary | Summary of casted votes
+
+**CastedVotesSummary:**
+| | Type | Description |
+|-|-|-|
+| totalvotes | int  | Total votes received |
+| optionsresult | voteoptionresult | Options info and votes received by each one |
+
+**Proposal voting status map:**
+| status | value | 
+|-|-|
+| Voting Active | 0 | 
+| Voting Finished | 1 | 
+| Voting not started | 2 |
```

Comment:
It should return one of the following answers:
```
const (
 ErrorStatusInvalid                     ErrorVoteStatusT = 0
 ErrorStatusNotStarted                  ErrorVoteStatusT = 1
 ErrorStatusStarted                     ErrorVoteStatusT = 2
 ErrorStatusFinished                    ErrorVoteStatusT = 3
 ErrorStatusDoesntExist                 ErrorVoteStatusT = 4
)
```

Also add a corresponding map to translate the numerical errors to human friendly string errors.